wiki/en/Running-a-Server.md

```diff
@@ -18,50 +18,39 @@ permalink: "/wiki/Running-a-Server"
 </details>
 
 
-
 ## Do I need to run a Server?
 
-The short answer is **no**. For various reasons, you should try to use Servers listed by the built in Directories, at least at first while getting your sound set up with the Jamulus Client. You may also be able to use a Server hosted by a third party. Doing so will save you the trouble of setting one up yourself.  
-
-Note that you can have a "private" session with other people on a Server by simply soloing each other. You will then not be able to hear anyone else if they enter your Server (note that this does not stop them from hearing you, using the chat function, or seeing your profile information).
+**No**. You can use the Servers listed by the built in Directories and use Jamulus without running a server.
 
-If you decide you cannot use any of the Servers listed by the built in Directories, you may be able to use a Server (either Registered in a Custom Directory, or Unregistered - see [Server Types](#server-types)) hosted by a third party. Doing so will save you the trouble of setting one up yourself.
+**Note:** You can even have a "private", uninterrupted session with other people on a Server by simply soloing each other. You will then not be able to hear anyone else if they enter your Server (note that this does not stop them from hearing you, using the chat function, or seeing your profile information).
 
-While setting up a Server isn't difficult, it's a good idea to read the following backround information to avoid some problems:
+If you don't want to use any of the Servers listed by the built in Directories, you may be able to use a Server (either Registered in a Custom Directory, or Unregistered - see [Server Types](#server-types)) hosted by a third party. Doing so will save you the trouble of setting one up yourself.
 
```

Comment:
I'd add line 30 (old) after line 27 (new).

Reply:
Hmm. Did that. But do we really need this line?

Reply:
I'm assuming line 29 (new now) is what I was saying should be kept.  Yes, I do still think it's needed - it's another reason that the answer to "Do I need to run a server?" is "No".  Not everyone is aware of third party providers.

Reply:
Added the third party provider text back in a slightly condensed form. I hope that's ok?